### PR TITLE
refactor: Create 'architect' folder

### PR DESCRIPTION
**Pull request by Roming22**
This commit is part of a larger set of commits with the goal of restructuring the repository.

Signed-off-by: Romain Arnaud <rarnaud@redhat.com>

### CODE REVIEW

**Review by Mo3m3n — CHANGES_REQUESTED** [review]
For each renamed/moved file or image we need to double check that it is not referenced elsewhere and update the reference.
Example: `docs/images/phase1.png` was moved to architect folder which broke the links in:
- ./README.md
- gitops/README.md
- architect/phase1.md

**Comment by Roming22** [comment]
> For each renamed/moved file or image we need to double check that it is not referenced elsewhere and update the reference. 

Done. Hopefully I did not miss any, but I fully expect that after moving everything around, there will be a few follow-up PRs to fine tune everything.

File: README.md [hunk 1]
```diff
@@ -26,43 +26,6 @@ Pipelines as Code is the veneer that brings to the users a great experience dire
...
-
-Tekton resource schemas are added only in kcp.
-
-Kubernetes workload clusters know nothing about Tekton. They only run resources like Deployments, Pods and Services.
-
-![Phase 2 flow](./docs/images/phase2.png)
-
 ## How do I start?
 
 ### Running in Kubernetes or OpenShift
```
Comment (bnallapeta) [inline README.md:31]: The content under this section says
You can deploy this PoC to your Kubernetes cluster with the `build.sh` and `run.sh` scripts in `ckcp` folder.
Please update this to 
You can setup Pipeline Service on your OpenShift cluster with the `openshift_dev_setup.sh` script in `ckcp` folder.

Comment (Roming22) [reply README.md:31]: Done

**Pull request merged**